internal/stores/contracts.go

```diff
@@ -224,7 +224,7 @@ func (s *SQLStore) ActiveContracts() ([]api.ContractMetadata, error) {
 	var dbContracts []dbContract
 	err := s.db.
 		Model(&dbContract{}).
-		Preload("Host.Announcements").
+		Preload("Host").
```

Comment:
do we need to preload the host here?

Reply:
Yeah we need some of that info to build the `api.ContractMetadata`.